CHANGELOG.md

```diff
@@ -10,8 +10,28 @@
   * The `tracing.type: jaeger` configuration, in `KafkaConnect`, `KafkaMirrorMaker`, `KafkaMirrorMaker2` and `KafkaBridge` resources, is not supported anymore.
   * The OpenTelemetry based tracing is the only available by using `tracing.type: opentelemetry`.
 * Added version fields to the `Kafka` custom resource status to track install and upgrade state
+* Support for infinite auto-restart of Kafka Connect connectors
 
 ### Changes, deprecations and removals
+
+* **The default behavior of the Kafka Connect connector auto-restart has changed.**
+  When the auto-restart feature is enabled in `KafkaConnector` or `KafkaMirrorMaker2` custom resources, it will not continue to restart the connectors forever and not stop after 7 restarts.
```

Comment:
```suggestion
  When the auto-restart feature is enabled in `KafkaConnector` or `KafkaMirrorMaker2` custom resources, it will now continue to restart the connectors indefinitely rather than stopping after 7 restarts, as previously.
```